vyuh_cli/lib/src/commands/create/utils/utils.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:mason/mason.dart';
+import 'package:path/path.dart' as path;
+import 'package:universal_io/io.dart';
+import 'package:vyuh_cli/src/commands/create/utils/logger_extension.dart';
+
+typedef MasonGeneratorFromBundle = Future<MasonGenerator> Function(MasonBundle);
+
+typedef MasonGeneratorFromBrick = Future<MasonGenerator> Function(Brick);
+
+final RegExp identifierRegExp = RegExp('[a-z_][a-z0-9_]*');
+final RegExp orgNameRegExp = RegExp(r'^[a-zA-Z][\w-]*(\.[a-zA-Z][\w-]*)+$');
+
+const String vyuhInfoText = '''
+
++----------------------------------------------------+
+| Looking for more features?                         |
+| We have an enterprise-grade solutions!             |
```

Comment:
solution --- singular

Also, we can add a line (---) after the title line

Reply:
Done